src/monexamine.cpp

```diff
@@ -234,7 +234,7 @@ void monexamine::mount_pet( monster &z )
     if( z.has_effect( effect_tied ) ) {
         z.remove_effect( effect_tied );
         if( z.tied_item ) {
-            g->u.i_add( *z.tied_item, 0 );
+            g->u.i_add( *z.tied_item, false );
```

Comment:
I'm inclined to believe there's no reason this shouldn't stack.
```suggestion
            g->u.i_add( *z.tied_item );
```

Reply:
just that one, or both?

Reply:
both 👍